exporter/clickhousetracesexporter/writer.go

```diff
@@ -167,6 +170,13 @@ func (w *SpanWriter) writeBatch(batch []*Span) error {
 			return err
 		}
 	}
+	if w.attributeKeyTable != "" {
```

Comment:
The serial insertions, one at a time, will affect the overall throughput. It wouldn't have been an issue if the batch consisted only attributes because they are relatively small, but here it is iterating on the whole span batch and preparing the insertion.

Reply:
We can merge the `writeTagKeyBatch` and `writeTagBatch` to reduce iterations.